### ISSUE TITLE
Filme werden nicht mehr erkannt unter XBMC

### ISSUE THREAD

**Issue by RalfEs73** [StatusQuo/filminfoorganizer#4]
Hallo,

ich nutze auf meinem XBMC als Scraper "Local Information only".
Damit wird die von Film Info! erstellte movie.nfo ausgelesen.
Seit dem Update von Film Info! auf 0.6.1.0 werden jedoch neue Filme nicht mehr erkannt. Also die Filme die eine neue movie.nfo bekommen haben. Ich testete das mit einem Film, bei dem ich die movie.nfo gelöscht habe und eine Neue mit der neuen Version von Film Info! erstellt habe. Auch diese wurde von XBMC nicht eingelesen.
Ist dieses Problem bereits bekannt?

**Comment by Dante999** [comment]
Hallo RalfEs73, 

Dieses Problem ist mir bei "XBMC Frodo" ebenfalls aufgefallen. 
Problem hierbei ist (meines Wissens), dass XBMC nicht mehr nach einer "movie.nfo" sucht, sondern nach einer "[Filmtitel].nfo"

Beispiel: 
Videodatei:        Ironman.mkv
geforderte nfo:  Ironman.nfo

und nicht mehr pauschal: movie.nfo

---

Selbes gilt übrigens auch für die Filmcover

Beispiel:                    Ironman.mkv
gefordertes Cover:    Ironman-poster.jpg

und nicht mehr pauschal: folder.jpg

---

Werden die Dateien so benannt, übernimmt sie XBMC eins-zu-eins. 

Falls interesse besteht, kann ich dir eine Batchdatei als vorläufiges Workaround anbieten, dass jeden einzelnen Ordner nach einer movie.nfo und folder.jpg durchsucht und entsprechend dem Namen der Video Datei umbenennt. . 

Gruß, 
Dante999

**Comment by StatusQuo** [comment]
Also ich habe mit dem xbmc und der Speicherung der Metadaten noch nie Probleme.
Habt ihr den Hacken bei "Filme in seperaten Ordnern" gemacht?

**Comment by RalfEs73** [comment]
Ja, der Hacken bei "Filme in separaten Ordnern" ist drin. Das mit dem Batch ist eine gute Idee. Hast du den bereits zur Hand?

Grüße,
Ralf

**Comment by Dante999** [comment]
@StatusQuo: welche XBMC Version benutzt du denn? Wäre es möglich, dass diese Version eine andere Benamung der Dateien fordert? 

Ich muss ehrlich gesagt gestehen, in XBMC selbst bin ich kein Profi, aber das mit der Benamung der Dateien habe ich bei mir ausgiebig getestet. 
Ich habe die NFO Dateien in den Filmordnern nachbearbeitet um zu erkennen ob xbmc auch wirklich die lokalen verwendet und nicht eigene scrapt, selbes git für die Filmcover. 

Ergänzung: Wenn ich mich recht erinnere, gilt die von mir genannte Benamung nur bei einzelnen Video Dateien. Liegt im Ordner eine DVD, also nur ein "VIDEO_TS" Ordner, funktioniert die "movie.nfo" problemlos.

@RalfEs73  Na klar, unter folgendem Link: https://github.com/Dante999/Film-Info_xbmc_workaround

**Comment by StatusQuo** [comment]
Also wir benutzen 13 jetzt vorher 12 halt immer ohne Probleme. Automatisches scrappen habe ich ausgeschaltet und deshalb gehe ich davon aus, dass das so funktioniert wie der Film Info! das macht.

**Comment by Dante999** [comment]
okay, das ist seltsam... Habe gerade noch einmal den "Ember Media Manager" ausprobiert, dieser nennt die Dateien (wenn als XBMC Version "Frodo" ausgewählt wird) genauso, wie ich vorgeschlagen habe...

Wäre es ohne größeren Aufwand möglich, dein Programm beim Einstellen des Media Centers für XBMC zu erweitern, damit man die Namensgebung beinflussen kann?
